src/main/java/io/github/railroad/Railroad.java

```diff
@@ -6,24 +6,25 @@
 import io.github.railroad.minecraft.ForgeVersion;
 import io.github.railroad.minecraft.MinecraftVersion;
 import io.github.railroad.project.ProjectManager;
-import io.github.railroad.project.ui.project.newProject.NewProjectPane;
 import io.github.railroad.project.ui.welcome.WelcomePane;
 import javafx.application.Application;
 import javafx.scene.Scene;
 import javafx.scene.input.KeyCode;
+import javafx.scene.layout.Pane;
+import javafx.stage.Screen;
 import javafx.stage.Stage;
 import okhttp3.OkHttpClient;
 
 import java.io.InputStream;
 import java.net.URL;
-import java.util.Objects;
-import java.util.concurrent.atomic.AtomicReference;
 
 public class Railroad extends Application {
     public static final OkHttpClient HTTP_CLIENT = new OkHttpClient();
     public static final Gson GSON = new GsonBuilder().setPrettyPrinting().disableHtmlEscaping().create();
     private static boolean DEBUG = false;
-    private final AtomicReference<NewProjectPane> newProjectPane = new AtomicReference<>();
+
+    public static Scene SCENE;
+    public static Stage WINDOW;
```

Comment:
These should be made `private` and getters used to access them, because since they are not `final`, they can otherwise be modified.